src/bump.ts

```diff
@@ -420,6 +422,12 @@ export async function publishBump(
           await createRelease(nv, headSha, changelog, isDev, isRc);
         }
       }
+      if (shouldCreateReleaseBranch) {
+        createBranch(
+          `refs/heads/release/${nextVersion.major}.${nextVersion.minor}`,
```

Comment:
Shouldn't we use here the `release-branches` configuration value to handle this? Or how do we want to handle the scenario if that configuration is set differently? Maybe disallow configuring both `release-branches` and `sdkver-create-release-branches`?

Reply:
That's a fair point; the reason this is hardcoded to `release/` is that we've (implicitly) specced that to be the pattern for SdkVer, but it still uses the cross-version-strategy `release-branches` configuration option. Since that option is a pattern, it isn't usable as a value here (e.g. `^rel(?:ease)?\/.+`). I'll think about a better way to make this a bit less ambiguous/more strict.

- [ ] Handle configurable release branch option.